GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -1,33 +1,43 @@
-﻿using GVFS.Common;
+﻿using System.Runtime.InteropServices;
+using GVFS.Common;
 using GVFS.Common.FileSystem;
 using GVFS.Common.Tracing;
-using GVFS.Platform.POSIX;
-
+using GVFS.Platform.POSIX;
+
 namespace GVFS.Platform.Mac
 {
     public partial class MacPlatform : POSIXPlatform
     {
         public MacPlatform()
             : base(
-                installerExtension: ".dmg")
+                installerExtension: ".dmg",
+                supportsGVFSService: true)
         {
-        }
-
+        }
+
         public override IDiskLayoutUpgradeData DiskLayoutUpgrade { get; } = new MacDiskLayoutUpgradeData();
         public override IKernelDriver KernelDriver { get; } = new ProjFSKext();
 
         public override string GetOSVersionInformation()
         {
             ProcessResult result = ProcessHelper.Run("sw_vers", args: string.Empty, redirectOutput: true);
             return string.IsNullOrWhiteSpace(result.Output) ? result.Errors : result.Output;
-        }
```

Comment:
What is the change here?

Reply:
likely the line endings. I've been editing this across Windows and Mac.